packages/edit-post/src/components/layout/index.native.js

```diff
@@ -129,10 +129,10 @@ class Layout extends Component {
 						parentHeight={ this.state.rootViewHeight }
 						style={ toolbarKeyboardAvoidingViewStyle }
 					>
+						{ showPageTemplatePicker && <__experimentalPageTemplatePicker /> }
```

Comment:
In #19106 I was also moving the picker to the bottom, although I guess it makes more sense to do it as part of this PR.